src/main/java/core/basesyntax/FileWork.java

```diff
@@ -8,7 +13,49 @@
  * Результат: web wide width world</p>
  */
 public class FileWork {
+    public static final String FIRST_WORD = "w";
+
     public String[] readFromFile(String fileName) {
-        return null;
+
+        File file = new File(fileName);
+        if (file.length() == 0) {
+            return new String[0];
+        }
+
+        String line = "";
```

Comment:
Use StringBuilder to avoid concatenation

Reply:
changed